src/cclib/parser/daltonparser.py

```diff
@@ -501,6 +515,9 @@ def extract(self, inputfile, line):
             self.set_attribute("charge", int(line.split()[-1]))
         if "@    Spin multiplicity and 2 M_S" in line:
             self.set_attribute("mult", int(line.split()[-2]))
+        # Dalton only has ROHF, no UHF
+            if self.mult != 1:
+                self.metadata["spintype"] = "ROHF"
```

Comment:
I don't think it's a good idea for cclib attributes to be constrained by CSX. How about we just have one fields called `method` and put all the methods used in the job in there? This would include both Hartree-Fock and post-HF methods. For example:
>>> print data1.metadata['methods']
>>> ['RHF', 'MP2', 'MP3']
>>> print data2.metadata['methods']
>>> ['UHF', 'CCSD(T)']

This would be more inclusive and extendable than creating a multitude of fields for different types of methods.

@berquist any thoughts?